pkg/imagestore/builder/builder.go

```diff
@@ -0,0 +1,81 @@
+package builder
+
+import (
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/deislabs/duffle/pkg/imagestore"
+	"github.com/deislabs/duffle/pkg/imagestore/ocilayout"
+	remote2 "github.com/deislabs/duffle/pkg/imagestore/remote"
```

Comment:
Same as for `builder2` - I guess you didn't want to shadow the `remote` variable, but this is rather counter-intuitive - agian, not something that should block the PR.

Reply:
Same issue again - I refactored a variable and didn't pay enough attention afterwards. Fixed.